### DISCUSSION TITLE
User management & landing page

### DISCUSSION THREAD

**Discussion by 404-fpv** [Attendize/Attendize#1049] [Ideas]
Hey,

I have just managed to get an install of Attendize up and running, it looks great so far.

For our use, there are two big stumbling blocks though that I could see as probable features that would be useful on a wider scale.

So for us, it would be great to have the ability to manage users in a bit of a different way, such as limit them to certain organizers, allowing us to delegate an organizer with a user which can only manage the organizations that have been assigned.
Also the ability to disable and/or delete a user is a massive security requirement also.

The second feature we would like to see would be a central landing page that could list all of the organizers (or ones assigned to the page) which would then allow us to point to that page and have customers able to see all events in a calendar like manner.

We are trying to use Attendize to manage a UK racing league for FPV wings and the idea is that each organizer would be a different regional club who will manage their own events. This is where our need for user management stems and then the need for a central landing page means all pilots are able to see all events coming up without needing to know the specific organizers url etc.

Many thanks for the hard work so far!

**Comment by justynpride** [comment]
Hi

There is a feature wanting to be merged which is #803 and #957. This would give you the manage users feature that you would want.
